client/blocks/upwork-banner/index.jsx

```diff
@@ -79,33 +72,22 @@ class UpworkBanner extends PureComponent {
 			return null;
 		}
 		return (
-			<ExternalLink
-				className="upwork-banner"
-				role="button"
-				style={ { backgroundColor: '#DAF5FC' } }
-				onClick={ this.onStartNowClick }
-				href="https://wordpress.com/built-by-wordpress-com/"
-			>
+			<>
 				<QueryPreferences />
-				<h1 className="upwork-banner__title">
-					{ translate( 'Let Our WordPress.com Experts Build Your Site!' ) }
-				</h1>
-				<p className="upwork-banner__description">
-					{ translate( 'You want the website of your dreams. Our experts can create it for you.' ) }
-				</p>
-				<Button className="upwork-banner__cta" compact primary={ this.props.primaryButton }>
-					{ translate( 'Find your expert' ) }
-				</Button>
-				<Button className="upwork-banner__close" onClick={ this.onDismissClick }>
-					<Gridicon icon="cross-small" size={ 18 } />
-				</Button>
-				<img
-					alt={ translate( 'Find your expert' ) }
-					width={ 390 }
-					className="upwork-banner__image"
-					src={ builderIllustration }
+				<UpsellNudge
+					forceDisplay //Upwork banner has its own logic for showing/hiding
+					className="upwork-banner"
+					callToAction={ translate( 'Find your expert' ) }
+					onClick={ this.onStartNowClick }
+					href="https://wordpress.com/built-by-wordpress-com/"
+					title={ translate( 'Let our WordPress.com experts build your site!' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Let Our WordPress.com Experts Build Your Site!' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/upwork-nudge)